components/job-form.js

```diff
@@ -47,6 +47,7 @@ export default function JobForm ({ item, sub }) {
         location: $location, remote: $remote, text: $text,
         url: $url, maxBid: $maxBid, status: $status, logo: $logo, hash: $hash, hmac: $hmac) {
         id
+        text
```

Comment:
This and the other similar changes are no longer needed. Added them in e129d85a but that was for the old approach.